config/eslint/.eslintrc-test

```diff
@@ -11,6 +11,8 @@ plugins:
 
 rules:
   mocha/no-exclusive-tests: 2
+  no-magic-numbers: 0
+  import/no-unresolved: 0 # We have custom `src` webpack prefix that is not node-resolvable
```

Comment:
Should we keep a separate `.eslintrc-demo` or just extend `.eslintrc-source` and add rules?